### PR TITLE
chore: adding linux questions

### PR DESCRIPTION
**Pull request by 201B153**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

For issue #16

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -674,6 +674,39 @@ const linuxQuiz = [
...
+      "We can use chown to change user and group simultaneously",
+    Link: "https://www.freecodecamp.org/news/linux-chmod-chown-change-file-permissions/",
+  },
+  {
+    Question:
+      "In Linux, how do you change permissions using absolute mode",
+    Answer: "using numbers and mathematical operators",
+    Distractor1: "using chmod",
+    Distractor2: "using chown",
+    Distractor3: "cannot change",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:705]: The answers and distractors don't show you how to change permissions using absolute mode.

Let's go with this change so it better fits the question.

```
    Answer: "chmod 451 file-name",
    Distractor1: "permissions 451 file-name",
    Distractor2: "change 451 file-name",
    Distractor3: "chown 451 file-name",
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -674,6 +674,39 @@ const linuxQuiz = [
...
+    Distractor1: "chmod user:group filename",
+    Distractor2: "chown user::group filename",
+    Distractor3: "chmod user::group filename",
+    Explanation:
+      "We can use chown to change user and group simultaneously",
+    Link: "https://www.freecodecamp.org/news/linux-chmod-chown-change-file-permissions/",
+  },
+  {
+    Question:
+      "In Linux, how do you change permissions using absolute mode",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:701]:
```suggestion
      "In Linux, what is the correct syntax for changing permissions using absolute mode?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

Also, you don't need to close a PR and open a new one when changes are requested. 
You can just push the new changes to this PR and it will be automatically be updated. 👍

File: src/data/linux-quiz.ts (hunk 1 above)
Comment (jdwilkin4) [reply src/data/linux-quiz.ts:705]: @201B153 
It looks like you forgot about this change

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Review by 201B153 — COMMENTED** [review]
Sorry!! abt earlier. I'm new to open source.
Is this okay now?

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by 201B153** [comment]
thanks for all the help.

**Pull request merged**